config/initializers/flipper.rb

```diff
@@ -0,0 +1,6 @@
+class CanAccessFlipperUI
+  def self.matches?(request)
+    current_user = request.env['warden'].user
+    current_user.present? && current_user.is_a?(AdminUser)
```

Comment:
I think this should be documented because we are imposing that the user should be of type `AdminUser`.